woodwork/tests/utils/test_read_file.py

```diff
@@ -116,3 +116,20 @@ def test_read_file(sample_df_pandas, tmpdir, filepath, exportfn, kwargs, pandas_
         schema_df = schema_df.head(kwargs["nrows"])
 
     pd.testing.assert_frame_equal(df, schema_df)
+
+
+def test_replace_empty_strings(tmpdir):
+    filepath = os.path.join(tmpdir, 'data.parquet')
+    content_type = 'application/parquet'
+
+    data = {
+        "double": ["<NA>", "6.2", "4.2", "3.11"],
+        "integer": ["<NA>", "6", "4", "3"],
+        "null": ["<NA>"] * 4,
```

Comment:
Per my other comment, I think we need to stick with the `"<NA>"` strings here since writing a `pd.NA` to parquet and then reading it back in results in values of `None` and not `"<NA>"` strings.

I will add some of the other values to this test though.

I'm not seeing any reason to skip `Int64` or `Float64` dtypes as those don't generate errors like the `boolean` and `string` columns do. @jeff-hernandez Do you know why `Int64` and `Float64` columns were skipped?

Reply:
Not sure why those dtypes were skipped. I think this function was copied directly from AML